### PR TITLE
Add background jobs

### PR DESCRIPTION
**Pull request by Sambok**
@Sbrady11 @heretoinfinity

### CODE REVIEW

File: app/channels/playground_channel.rb [hunk 1]
```diff
@@ -6,6 +6,19 @@ def subscribed
...
   end
 
-  def create(opts)
+  def create_rspec_code(opts)
+    @rpec_code = RspecCode.create(
+      rspec: opts.fetch('rspec')
+    )
+  end
+
+  def create_rspec_code(opts)
```
Comment (Sbrady11) [inline app/channels/playground_channel.rb]: This should be `def create_ruby_code(opts)` :)

Comment (Sbrady11) [reply app/channels/playground_channel.rb]: cmon man this is simple stuff!!!!!! 🤠

File: app/jobs/run_rspec_event_broadcast_job.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class RubyCodeCreationEventBroadcastJob < ApplicationJob
+  queue_as :default
+  
+  def perform
+    ActionCable.server.broadcast(
+      'playgrounc_channel',
```
Comment (Sbrady11) [inline app/jobs/run_rspec_event_broadcast_job.rb]: spell check

File: app/channels/playground_channel.rb [hunk 3]
```diff
@@ -6,6 +6,19 @@ def subscribed
   def unsubscribed
   end
 
-  def create(opts)
+  def create_rspec_code(opts)
+    @rpec_code = RspecCode.create(
```
Comment (heretoinfinity) [inline app/channels/playground_channel.rb]: 's' missing. @rspec_code

File: app/jobs/ruby_code_creation_event_broadcast_job.rb [hunk 4]
```diff
@@ -1,3 +1,10 @@
 class RubyCodeCreationEventBroadcastJob < ApplicationJob
```
Comment (Sbrady11) [inline app/jobs/ruby_code_creation_event_broadcast_job.rb]: same class error

File: app/channels/playground_channel.rb [hunk 5]
```diff
@@ -6,6 +6,19 @@ def subscribed
...
\ No newline at end of file
+
+  def create_ruby_code(opts)
+    @ruby_code = RubyCode.create(
+      ruby: opts.fetch('ruby')
+    )
+  end
+
+  def run_rspec
+    RubyCodeCreationEventBroadcastJob.perform_now
```
Comment (Sbrady11) [inline app/channels/playground_channel.rb]: Not running rspeccreationeventbroadcastjob

Comment (Sbrady11) [reply app/channels/playground_channel.rb]: `rspecCreationEventBroadcastJob`

**Pull request merged**